.github/workflows/license_tests.yml

```
@@ -5,8 +5,9 @@ on:
   pull_request:
     branches:
       - master
+
 jobs:
   license_tests:
     uses: neongeckocom/.github/.github/workflows/license_tests.yml@master
     with:
-      packages-exclude: '^(pyklatchat-client|tqdm).*'
\ No newline at end of file
+      packages-exclude: '^(klatchat|neon|tqdm|RapidFuzz|typing_extensions).*'
```

`klatchat` should be replaced with `pyklatchat` to resolve the license test failures. Alternatively, we could rename this package to `klatchat-client`

```suggestion
      packages-exclude: '^(pyklatchat|neon|tqdm|RapidFuzz|typing_extensions).*'
```